manifests/d/DashoGames/PiozaGameLauncher/0.4.0/DashoGames.PiozaGameLauncher.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DashoGames.PiozaGameLauncher
+PackageVersion: 0.4.0
+InstallerLocale: pl-PL

```

Comment:
```suggestion
```
The installer is not specific to the `pl-PL` locale. If this is included, users of other locales may not be able to install the software